main.js

```diff
@@ -0,0 +1,108 @@
+//Players and Status
+let playerX = 'X'
+let playerO = 'O'
+currentPlayer = playerX
+let message = document.getElementById('status')
+//--------------------------------------------------------------------------------------
+//Start Game 1 Player
+let startButton = document.getElementById("start1")
+startButton.addEventListener('click', activateSquare)
+
+function activateSquare() {
+    c0.addEventListener('click', event), c1.addEventListener('click', event), c2.addEventListener('click', event), c3.addEventListener('click', event), c4.addEventListener('click', event), c5.addEventListener('click', event), c6.addEventListener('click', event), c7.addEventListener('click', event), c8.addEventListener('click', event)
+    message.innerHTML = 'Current Player X'
+}
+//Start Game 2 Player
+let startButton2 = document.getElementById("start2")
+startButton2.addEventListener('click', activateSquare2)
+
+function activateSquare2() {
+    c0.addEventListener('click', event), c1.addEventListener('click', event), c2.addEventListener('click', event), c3.addEventListener('click', event), c4.addEventListener('click', event), c5.addEventListener('click', event), c6.addEventListener('click', event), c7.addEventListener('click', event), c8.addEventListener('click', event)
+    message.innerHTML = 'Current Player X'
+}
+//Switches between X and O and draws the player symbol, and blocks square in 1 Player mode and 2 Player mode
+function event(event) {
+
+    if (event.target.innerHTML) {
+        message.innerHTML = 'SQUARE IS OCCUPIED'
+    } else {
+        event.target.innerHTML = currentPlayer
+        //
+        if (currentPlayer === playerX) {
+            currentPlayer = playerO
+            message.innerHTML = 'Current Player O'
+            winningLine()
+        } else {
+            currentPlayer = playerX
+            message.innerHTML = 'Current Player X'
+            winningLine()
+        }
+    }
+}
+//Discovers a winning combination in 1 Player mode and 2 Player mode
+function winningLine() {
+    for (let winningArrays of winningconditons) {
+        let firstCell = winningArrays[0].innerHTML;
+        let secondCell = winningArrays[1].innerHTML;
+        let thirdCell = winningArrays[2].innerHTML;
+        if (firstCell !== "") {
+            if (firstCell === secondCell && secondCell === thirdCell) {
+                winner = true;
+                message.innerHTML = 'WINNER'
+                winningArrays[0].style.backgroundColor = "blue"
+                winningArrays[1].style.backgroundColor = "blue"
+                winningArrays[2].style.backgroundColor = "blue"
+                deactivateSquares()
+                clearBoard()
+                break;
+            }
+        }
+    }
+    draw()
+}
+//Clears board game and reset board after 3 seconds
+function clearBoard() {
+    window.setTimeout(reloadDocument, 3000);
+}
+
+function reloadDocument() {
+    document.location.reload();
```

Comment:
This works for now, but might not be the best way to reset the board from a UX/UI point of view.